docs/sign-up.rst

```diff
@@ -12,10 +12,17 @@ As an organization administrator, you can also manage other CrateDB Cloud users
 as well as launch CrateDB Cloud clusters and other CrateDB Cloud products.
 
 We currently support two signup methods. However, only one authentication
-method, AzureAD, allows you to deploy clusters on CrateDB Cloud. For a more
-detailed tutorial guiding you through the Azure signup process through to
-starting up a cluster, see the :ref:`tutorial on deploying a cluster from
-scratch <deploy-azure-to-cluster>`.
+method, AzureAD, allows you to deploy clusters on CrateDB Cloud through
+Microsoft Azure Marketplace. For our other cloud provider, AWS, either method
+will work.
```

Comment:
we had to change this on short notice. AzureAD only for Azure MP. and username&pwd (cognito) only for AWS MP

Reply:
Yeah, changed that now.